### PR TITLE
feat(cxl-ui): vaadin-notification integration

### PR DESCRIPTION
**Pull request by saas786**
ref:
https://github.com/conversionxl/aybolit/issues/84
https://github.com/conversionxl/jobs-theme/issues/93

### CODE REVIEW

File: packages/cxl-ui/package.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
     "@vaadin/vaadin-progress-bar": "^2.0.0-alpha2",
     "@vaadin/vaadin-tabs": "^4.0.0-alpha4",
     "@vaadin/vaadin-themable-mixin": "^1.6.2",
+    "@vaadin/vaadin-notification": "^1.6.1",
```
Comment (lkraav) [inline packages/cxl-ui/package.json:24]: Go with latest P3 alphas for everything now https://github.com/vaadin/vaadin-notification/releases

**Review by lkraav — CHANGES_REQUESTED** [review]

File: packages/cxl-ui/package.json (hunk 1 above)
Comment (saas786) [reply packages/cxl-ui/package.json]: > Go with latest P3 alphas for everything now https://github.com/vaadin/vaadin-notification/releases

Bumped the version to **2.0.0-alpha1**

**Comment by lkraav** [comment]
Hmm, I see. I was actually more expecting to find out how pure vaadin-notification could be integrated into Gravity Forms :thinking: 

We could use seeing an example of actual usage to understand how useful of a solution this is.

For example, I'm not clear on why we need to wrap vaadin-notification at all, at least to start.

**Comment by saas786** [comment]
> Hmm, I see. I was actually more expecting to find out how pure vaadin-notification could be integrated into Gravity Forms 🤔
> 
> We could use seeing an example of actual usage to understand how useful of a solution this is.
> 
> For example, I'm not clear on why we need to wrap vaadin-notification at all, at least to start.

Its close to pure `vaadin-notification` integration, there are various reasons I choose to use LitElement.

a) While working on `vaadin-dialog`, I faced various issues using `<template>`, so instead opted to use
`renderer` 
ref: https://cdn.vaadin.com/vaadin-notification/2.0.0-alpha1/#/elements/vaadin-notification#description

b) Instead of handling `renderer` in PHP (theme etc), I think should handle it in component itself.

c) Could utilize `theme` for `<vaadin-notification-container>` (by default not implemented), need to explore how I can achieve it, so can use inline notifications as well instead of overlay (default).

d) By default close button is not provided, so preferred to handle it inside the component, over maintaining it in PHP.

**Comment by lkraav** [comment]
> d) By default close button is not provided, so preferred to handle it inside the component, over maintaining it in PHP.

OK, but are you able to post a wp-theme-lib PR for Gravity Forms integration for vanilla vaadin-notification? This will massively clarify what issues are real, and what are not.

Next immediate target would be Sensei (course, lesson, quiz screens), then WooCommerce (checkout).

**Comment by saas786** [comment]
> > d) By default close button is not provided, so preferred to handle it inside the component, over maintaining it in PHP.
> 
> OK, but are you able to post a wp-theme-lib PR for Gravity Forms integration for vanilla vaadin-notification? This will massively clarify what issues are real, and what are not.
> 
> Next immediate target would be Sensei (course, lesson, quiz screens), then WooCommerce (checkout).

Do you suggest, I should move this: https://github.com/conversionxl/jobs-theme/commit/39c673349f3181ccbdc7ed1eab09d11af7b4ab61

to `wp-theme-lib`?

**Comment by lkraav** [comment]
> to `wp-theme-lib`?

Yes, this will be a central cross-app functionality.

**Comment by lkraav** [comment]
OK great stuff here, I had some additional findings to push to https://github.com/conversionxl/aybolit/compare/master...conversionxl:leho-vaadin-notification?expand=1 and I'll integrate it all tomorrow. This can quite likely go live quickly.

Looking forward to Sensei message upgrade.

**Comment by saas786** [comment]
> Sensei message

I don't have it on local install, how I can preview it on warmpress? Screenshots of relevant message(s) will be helpfu.

**Pull request closed without merging**